package/sbp_protocol/src/sbp_router_smoothpose.yml

```diff
@@ -143,14 +143,14 @@ ports:
       - dst_port: SBP_PORT_SETTINGS_DAEMON
         filters:
           - { action: ACCEPT }
-      - dst_port: SBP_PORT_SETTINGS_CLIENT
-        filters:
-          - { action: ACCEPT, prefix: [0x55, 0xAF, 0x00] } # Settings write response
-          - { action: REJECT }
+#      - dst_port: SBP_PORT_SETTINGS_CLIENT
```

Comment:
this a daemon sending a message to itself, pretty sure this is unnecessary

Reply:
this is the port used by any settings client, I agree it's a bit odd but needed for the watchers to track settings changes. will need to revert

Reply:
fixed, added a comment to explain what's going on